hp_list/Makefile

```diff
@@ -1,6 +1,11 @@
 all:
 	$(CC) -Wall -o list main.c -lpthread -g -fsanitize=thread
 
+# The RUNTIME_STAT allow to show the runtime operation state,
+# such as "rtry" is the number of retries in the __list_find function.
+analyze:
```

Comment:
Please rebase the latest `master` branch.
I have refactored the build system, and you can simply add `analyze` target as following:
```
analyze: CFLAGS += RUNTIME_STAT
analyze: $(BIN)
```

Then, `make analyze` will work for you.

Reply:
Use `git rebase` instead of `git merge`!